Gems/Multiplayer/Code/multiplayerstats_api_files.cmake

```diff
@@ -7,6 +7,7 @@
 #
 
 set(FILES
+    Include/Multiplayer/MultiplayerMetrics.h
```

Comment:
Probably want some heuristics written down somewhere about what the difference is between MultiplayerMetrics.h, MultiplayerStats.h, and MultiplayerPerformanceStats.h, because honestly I'm not 100% where I would put what if I were adding new metrics.